pkgs/by-name/ne/neovide/package.nix

```diff
@@ -94,9 +94,14 @@ rustPlatform.buildRustPackage.override { stdenv = clangStdenv; } rec {
 
       wrapProgram $out/bin/neovide \
         --prefix LD_LIBRARY_PATH : ${libPath}
+        --prefix PATH : ${lib.makeBinPath [ neovim python3 ]}
     '';
 
-  postInstall = ''
+  postInstall = if stdenv.isDarwin then ''
+    mkdir -p $out/Applications
+    cp -r extra/osx/Neovide.app $out/Applications
+    ln -s $out/bin/.neovide-wrapped $out/Applications/Neovide.app/Contents/MacOS/neovide
```

Comment:
You created a wrapped neovide, but you use the unwrapped one, why?

Reply:
My mistake, I misinterpreted [this comment](https://github.com/NixOS/nixpkgs/pull/319061#pullrequestreview-2110944294) to mean I was not using the wrapped executable in the .app bundle. I then I thought `.neovide-wrapped` would be the wrapped one, as its name suggested. But I see now, by inspecting the `neovide` executable, that it's just a script that exec `.neovim-wrapped`.